.github/workflows/gitlab.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v2.4.0 # REMEMBER to also update in .gitlab-ci.yml
+          triggered-ref: lukasp/batching-tests-long # REMEMBER to also update in .gitlab-ci.yml
```

Comment:
Is this just for testing now?

Reply:
I wanted to prove that it actually does something, so created accompanying PR in libtelio-build - https://bucket.digitalarsenal.net/low-level-hacks/vpn/client/libtelio-build/-/merge_requests/265 which filters for `batching` marked tests.

The regular `natlab` jobs wouldn't take this job.

Ideally I might revert it back before merging but probably no harm in even leaving it :)

Reply:
I've reverted those back